### PR TITLE
add copy text and download file

### PR DESCRIPTION
**Pull request by mytechtip**

### CODE REVIEW

File: src/ts/components/account/GeneratePhrase.tsx [hunk 1]
```diff
@@ -14,37 +14,67 @@ interface IGeneratePhraseProps extends StateProps, DispatchProps, RouteComponent
...
 
+  copyText = () => {
+    const el = document.createElement('textarea')
+    el.value = this.state.mnemonic
+    el.setAttribute('readonly', '')
+    el.style.position = 'absolute'
+    el.style.left = '-9999px'
+    document.body.appendChild(el)
+    el.select()
+    document.execCommand('copy')
```
Comment (ttaoS) [inline src/ts/components/account/GeneratePhrase.tsx]: does it exec `copy` on whole document?

Comment (mytechtip) [reply src/ts/components/account/GeneratePhrase.tsx]: it only copies the selected texts

File: src/ts/components/account/GeneratePhrase.tsx [hunk 2]
```diff
@@ -14,37 +14,67 @@ interface IGeneratePhraseProps extends StateProps, DispatchProps, RouteComponent
...
+    document.body.removeChild(el)
+
+    this.setState({message: t('copyTextMessage')})
```
Comment (ttaoS) [inline src/ts/components/account/GeneratePhrase.tsx]: I can't fine `copyTextMessage` in message. Are we plan to display something like `copying...` ?

File: src/ts/components/account/GeneratePhrase.tsx [hunk 3]
```diff
@@ -14,37 +14,67 @@ interface IGeneratePhraseProps extends StateProps, DispatchProps, RouteComponent
...
+    setTimeout(()=> {
+      this.setState({message: ''})
+    }, 3000)
```
Comment (ttaoS) [inline src/ts/components/account/GeneratePhrase.tsx]: 3 sec for copy?

Comment (mytechtip) [reply src/ts/components/account/GeneratePhrase.tsx]: Oh, I missed that message. Will add it. Basically it just displays a copy success message for 3 sec

File: src/ts/components/account/GeneratePhrase.tsx [hunk 4]
```diff
@@ -14,37 +14,67 @@ interface IGeneratePhraseProps extends StateProps, DispatchProps, RouteComponent
...
+  }
+
+  downloadFile = () => {
+    var element = document.createElement('a');
+    element.setAttribute('href',
+        'data:text/plain;charset=utf-8,' + encodeURIComponent(this.state.mnemonic))
+    element.setAttribute('download', 'secret-phrase.txt')
```
Comment (ttaoS) [inline src/ts/components/account/GeneratePhrase.tsx]: we will handle secret-phrase.txt generation later, right?

Comment (ttaoS) [reply src/ts/components/account/GeneratePhrase.tsx]: don't worry. i think i know how it works now.

**Pull request merged**